pypeman/exceptions.py

```diff
@@ -0,0 +1,21 @@
+class EndChanProcess(StopAsyncIteration):
+    """
+    A custom excption to tell that the channel reach the end
```

Comment:
exception ...  
reach -> reached

at moment -> currently / at the moment

after him -> after it

Reply:
done